analyse_purchases_demo.ipynb

```diff
@@ -262,7 +262,7 @@
         "        segment_df[numerator_column] = segment_df[numerator_column].replace(np.nan, -100)\n",
         "        segment_df[denominator_column] = segment_df[denominator_column].replace(np.nan, 100)\n",
         "        # selecting columns to maintain standard format\n",
-        "        segment_df = segment_df[['segment_name', numerator_column, metric_column_change_percentage]]\n",
+        "        segment_df = segment_df[['segment_name', numerator_column,denominator_column, metric_column_change_percentage]]\n",
```

Comment:
Add explanation for why this column was added to this data frame

Reply:
This column was added to maintain the information about yesterday's purchase value for that segment of that metric. 
a comment will be incorporated in the next version.